utils/dateUtils.ts

```diff
@@ -211,19 +211,23 @@ export function isPast7Days(date: Date): boolean {
   )
 }
 
-// Function to check if the date is older than 7 days
-export function isOlderThan7Days(date: Date): boolean {
-  const sevenDaysAgo = subDays(new Date(), 7)
-  return date < startOfDay(sevenDaysAgo)
+// Function to check if the date is from 1 month ago up to the start of this week (excluding this week)
+export function isPastMonth(date: Date): boolean {
+  const oneMonthAgo = subMonths(new Date(), 1)
+  const startOfThisWeek = startOfWeek(new Date())
+  
+  return (
+    isWithinInterval(date, {
+      start: oneMonthAgo,
+      end: startOfThisWeek
+    }) &&
+    isBefore(date, startOfThisWeek) // Ensures date is before the start of this week
+  )
 }
 
-// Function to check if the date is within the past 24 hours but not today
-export function isLast24Hours(date: Date): boolean {
-  const now = new Date()
-  const twentyFourHoursAgo = subHours(now, 24)
-  return isWithinInterval(date, { 
-    start: twentyFourHoursAgo, 
-    end: now 
-  }) && !isToday(date)
+// Function to check if the date is older than 1 month 
+export function isOlder(date: Date): boolean {
```

Comment:
Can you rename this be a little more descriptive? `isOlderThanOneMonth` would be fine - `isOlder` is just a little vague

Reply:
yes!!